tests/soter/nist-sts/makefile

```diff
@@ -1,94 +1,51 @@
-CC = /usr/bin/gcc
-GCCFLAGS = -c -Wall
-ROOTDIR = .
-SRCDIR = $(ROOTDIR)/src
-OBJDIR = $(ROOTDIR)/obj
-VPATH  = src:obj:include
+#
+# Copyright (c) 2019 Cossack Labs Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
-OBJ = $(OBJDIR)/assess.o $(OBJDIR)/frequency.o $(OBJDIR)/blockFrequency.o \
-      $(OBJDIR)/cusum.o $(OBJDIR)/runs.o $(OBJDIR)/longestRunOfOnes.o \
-      $(OBJDIR)/serial.o $(OBJDIR)/rank.o $(OBJDIR)/discreteFourierTransform.o \
-      $(OBJDIR)/nonOverlappingTemplateMatchings.o \
-      $(OBJDIR)/overlappingTemplateMatchings.o $(OBJDIR)/universal.o \
-      $(OBJDIR)/approximateEntropy.o $(OBJDIR)/randomExcursions.o \
-      $(OBJDIR)/randomExcursionsVariant.o $(OBJDIR)/linearComplexity.o \
-      $(OBJDIR)/dfft.o $(OBJDIR)/cephes.o $(OBJDIR)/matrix.o \
-      $(OBJDIR)/utilities.o $(OBJDIR)/generators.o $(OBJDIR)/genutils.o
+CC ?= /usr/bin/gcc
```

Comment:
but it's gonna work with clang, right? 🤔

Reply:
Yes, it works with clang as well. You just have to override this default:

```
make test CC=clang
```

Reply:
And we do override it, for example, for macOS:

https://github.com/cossacklabs/themis/blob/d0e70c18637bf29efb3fed315c3e21b0f659ddc3/Makefile#L191-L196

(Yes, it's called `gcc` but that's Clang on the inside.)

Reply:
so basically on macOS I don't need to do any extra work :) yey!